README.md

```diff
@@ -16,6 +16,7 @@ SystemLink service or collection of related services:
 | Message | [NationalInstruments.SystemLink.Clients.Message](https://github.com/ni/systemlink-client-docs/wiki/Message) | [![NuGet](https://img.shields.io/nuget/v/NationalInstruments.SystemLink.Clients.Message.svg)](https://www.nuget.org/packages/NationalInstruments.SystemLink.Clients.Message/)
 | Tag     | [NationalInstruments.SystemLink.Clients.Tag](https://github.com/ni/systemlink-client-docs/wiki/Tag) | [![NuGet](https://img.shields.io/nuget/v/NationalInstruments.SystemLink.Clients.Tag.svg)](https://www.nuget.org/packages/NationalInstruments.SystemLink.Clients.Tag/)
 | Test Monitor | [NationalInstruments.SystemLink.Clients.TestMonitor](https://github.com/ni/systemlink-client-docs/wiki/TestMonitor) | [![NuGet](https://img.shields.io/nuget/v/NationalInstruments.SystemLink.Clients.TestMonitor.svg)](https://www.nuget.org/packages/NationalInstruments.SystemLink.Clients.TestMonitor/)
+| Asset Mangement | [NationalInstruments.SystemLink.Clients.AssetManagement](https://github.com/ni/systemlink-client-docs/wiki/AssetManagement) | [![NuGet](https://img.shields.io/nuget/v/NationalInstruments.SystemLink.Clients.AssetManagement.svg)](https://www.nuget.org/packages/NationalInstruments.SystemLink.Clients.AssetManagement/)
```

Comment:
It would be nice to maintain alphabetic ordering. Though we may want to wait to add this here until the package has been published and has documentation on the wiki.